README.md

```diff
@@ -1,28 +1,27 @@
-# stoa
+# Stoa
 API server for Agora
 
+# Build instructions
+Install Node.js (you might want to use a newer version)
 
-
-## Build instructions
-
-# Install the node.Js >= 12.18.0 (includes npm 6.14.4) (you might want to use a newer version)
-
-
-# Building on Ubuntu
-# Ubuntu is already nodejs installed.
+## Building on Ubuntu
+```sh
 $ sudo apt-get install nodejs
 $ sudo apt-get install npm
 $ npm ci
```

Comment:
That's for testing, not building, no ? Should be `npm build`

Reply:
`npm ci` This command is similar to npm install.
Install dependent packages based on package-lock.json.

Reply:
I see, thanks

Reply:
The `npm build` acts as an integrated file with a web pack, but I'm thinking if I need it here.